front/src/modules/people/components/__stories__/PhoneEditableField.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { BrowserRouter } from 'react-router-dom';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { PhoneCellEdit } from '@/ui/table/editable-cell/type/components/PhoneCellEdit';
```

Comment:
@mustajab-ikram it's great that you are adding test. As this component is located in @/ui/table/editable-cell, storybook tests should be in UI/Table/Editable-Cell